docs/source/platform/operation-registry.md

```diff
@@ -66,26 +66,47 @@ abc123  <service>     current
 
 > If you encounter any errors, refer to the _**Troubleshooting**_ section below.
 
-**3. Register operations from the client bundle.**
+**3. Configure client reporting in the client bundle.**
 
-Now we'll use `apollo client:push` to locate operations within the client codebase and upload a manifest of those operations to Apollo operation registry. Once Apollo Server has been configured to respect the operation registry, only operations which have been included in the manifest will be permitted.
+The client must identity itself so we can match operations with a client and version. See the [Identifying clients](../platform/client-awareness.html#Setup) documentation to configure `ApolloClient` to identity itself. The following is an example:
```

Comment:
I don't think clients _must_ identify themselves to use the operation registry. It's a nice-to-have, but not required from my understanding